tests/moduletool/test_build.py

```diff
@@ -105,47 +123,59 @@ def test_build_v2_module(
 
     if not set_path_argument:
         monkeypatch.chdir(module_copy_dir)
-    run_module_build_soft(module_copy_dir, set_path_argument, byte_code=byte_code)
+    run_module_build_soft(module_copy_dir, set_path_argument, byte_code=byte_code, distribution=distribution)
 
     dist_dir = os.path.join(module_copy_dir, "dist")
     dist_dir_content = os.listdir(dist_dir)
     assert len(dist_dir_content) == 1
-    wheel_file = os.path.join(dist_dir, dist_dir_content[0])
-    assert wheel_file.endswith(".whl")
+    package_file = os.path.join(dist_dir, dist_dir_content[0])
+    if distribution == "wheel":
+        assert package_file.endswith(".whl")
+    else:
+        assert package_file.endswith(".tar.gz")
 
     extract_dir = os.path.join(tmpdir, "extract")
-    with zipfile.ZipFile(wheel_file) as zip:
-        zip.extractall(extract_dir)
-
-    if byte_code:
-        assert "linux_x86_64" in wheel_file
+    if distribution == "wheel":
+        with zipfile.ZipFile(package_file) as zip:
+            zip.extractall(extract_dir)
     else:
-        assert "none-any" in wheel_file
+        with tarfile.open(package_file) as tar:
+            tar.extractall(extract_dir)
+            # member_path = pathlib.Path(tar.getnames()[0])
+            # extract_dir = str(pathlib.Path(extract_dir).relative_to(member_path.parts[0]))
```

Comment:
leftover code?

Reply:
It seems I forgot to push my latest commits.